guides/source/active_record_querying.md

```diff
@@ -1821,6 +1821,21 @@ Client.limit(1).pluck(:name)
 # => ["David"]
 ```
 
+You should also know that using `pluck` will trigger eager loading if the relation object contains include values, even if the eager loading is not necessary for the query. For example:
```

Comment:
Would you mind prefix this with `NOTE:`?

(Looks like we need support for NOTE block though ... maybe I can add one 🤔)

Reply:
I will add the prefix. Although I totally support the idea of NOTE block as well.